components/dash-core-components/src/fragments/Math.react.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+
+export default class DashMath extends Component {
+    constructor(props) {
+        super(props);
+        this.span_element = React.createRef();
+    }
+
+    componentDidMount() {
+        this.renderMath();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (
+            prevProps.tex !== this.props.tex ||
+            prevProps.inline !== this.props.inline
+        ) {
+            this.renderMath();
+        }
+    }
+
+    renderMath() {
+        if (window.MathJax && window.MathJax.typeset) {
```

Comment:
```suggestion
        if (window.MathJax?.typeset) {
```
Living in the modern JS world over here, not like plotly.js 😅